packages/default-plugins/plugin-patches/io.github.jackgruber.backup.diff

```diff
@@ -0,0 +1,74 @@
+diff --git a/src/sevenZip.ts b/src/sevenZip.ts
+index ef2a527..253d21a 100644
+--- a/src/sevenZip.ts
++++ b/src/sevenZip.ts
+@@ -1,21 +1,21 @@
+ // https://sevenzip.osdn.jp/chm/cmdline/exit_codes.htm
+ // https://sevenzip.osdn.jp/chm/cmdline/commands/index.htm
+ import * as _7z from "node-7z";
+-import * as sevenBin from "7zip-bin";
+-import * as path from "path";
++const sevenBin = require("7zip-bin");
+ import { exec } from "child_process";
+ import joplin from "api";
+ 
+-export let pathTo7zip = sevenBin.path7za;
+-
+-export namespace sevenZip {
+-  export async function updateBinPath() {
+-    pathTo7zip = path.join(
+-      await joplin.plugins.installationDir(),
+-      "7zip-bin",
+-      pathTo7zip
+-    );
+-  }
++ export let pathTo7zip = sevenBin.path7za;
++
++ export namespace sevenZip {
++   export async function updateBinPath() {
++    // Not necessary with 7zip required from Joplin
++    // pathTo7zip = path.join(
++    //   await joplin.plugins.installationDir(),
++    //   "7zip-bin",
++    //   pathTo7zip
++    // );
++   }
+ 
+   export async function setExecutionFlag() {
+     if (process.platform !== "win32") {
+diff --git a/webpack.config.js b/webpack.config.js
+index 34a1797..fbf655b 100644
+--- a/webpack.config.js
++++ b/webpack.config.js
+@@ -175,6 +175,12 @@ const baseConfig = {
+ 		],
+ 	},
+ 
++	// We support requiring 7zip-bin through require('7zip-bin')
++	externalsType: 'commonjs',
++	externals: {
++		'7zip-bin': { commonjs: '7zip-bin' },
++	},
++
+ 	// 7zip-bin uses __dirname to determine the path to bundled binaries.
+ 	// In Joplin, __dirname points to the plugin directory (for example
+ 	// /tmp/.mount_Joplinf4CQVb/resources/app.asar/services/plugins/),
+@@ -200,15 +206,9 @@ const pluginConfig = { ...baseConfig, entry: './src/index.ts',
+ 		path: distDir,
+ 	},
+ 	plugins: [
+-		new CopyPlugin({
+-			patterns: [
+-				{
+-					from: '**/*',
+-					context: path.resolve(__dirname, 'node_modules','7zip-bin'),
+-					to: path.resolve(__dirname, 'dist/7zip-bin/'),
+-				},
+-			]
+-		}),
++		// Removed a CopyPlugin (added by Simple Backup, not necessary when using
++		// Joplin's built-in 7zip)
++
+ 		new CopyPlugin({
+ 			patterns: [
+ 				{
```

Comment:
Sorry I think you mentioned something about this before, but is there any reason we can't submit this patch as a pull request?